src/bin/watch.rs

```diff
@@ -1,34 +1,36 @@
 #![deny(warnings)]
-
-// TODO support reading from standard input
-
 extern crate extra;
 extern crate termion;
 
 use std::{cmp, str, thread};
 use std::env::args;
-use std::io::{self, Write, Read};
-use std::process::{self, Command, Stdio};
+use std::io::{self, Read, Write};
+use std::process::{Command, Stdio};
 use std::time::Duration;
 
+use extra::io::fail;
 use extra::option::OptionalExt;
 
 use termion::{async_stdin, clear, cursor, style, terminal_size};
 use termion::raw::IntoRawMode;
 
-static MAN_PAGE: &'static str = /* @MANSTART{watch} */ r#"
```

Comment:
You need to leave in this `@MANSTART{watch}` comment. It is used by `docgen` to generate man pages.